CMakeLists.txt

```
@@ -230,6 +230,7 @@ option(USE_LITE_PROTO "Use lite protobuf instead of full." OFF)
 option(USE_LMDB "Use LMDB" OFF)
 option(USE_MAGMA "Use MAGMA" ON)
 option(USE_METAL "Use Metal for Caffe2 iOS build" ON)
+option(USE_ONNX "Use Onnx" OFF)
```

ONNX should be capitalized, as it's an acronym
```suggestion
option(USE_ONNX "Use ONNX" OFF)
```

---

Default value of this option should be dependent on `BUILD_PYTHON` - I.e. in case of PyTorch builds we want to keep it enabled, but for libtorch (i.e. when `BUILD_PYTHON` is `OFF` ) it should be disabled

---

Thanks for the feedback @malfet, I was curious if you have any input about this comment from @BowenBao ?

> You might need to move onnx related files around in
> 
> https://github.com/pytorch/pytorch/blob/440b7d52793be046354c03521295ab7f2d3969f2/build_variables.bzl#L637
> 
> out from the libtorch ones. I'm not an expert in the bzl build so unclear how to restructure them while not affecting standard pytorch build. @malfet would love some advice here.

---

heads up that I'm still stuck on this questions by @BowenBao , @malfet , hoping to get some input so I take the PR in the right direction. thanks!